### PR TITLE
Retrieve random bitfield 2

### PR DESCRIPTION
**Pull request by philipstanislaus**
Replaces #416

### CODE REVIEW

**Comment by philipstanislaus** [comment]
@musnit can we merge this?

File: ethereum/test/test_verification.js [hunk 1]
```diff
@@ -58,13 +69,16 @@ describe.skip("Verification tests", function () {
...
-      );
-      this.app = await MockApp.new();
+      // this.channel = await BasicInboundChannel.new(this.lightClientBridge.address,
+      //   { from: owner }
+      // );
+      // this.app = await MockApp.new();
     });
 
     it("should successfully verify a commitment", async function () {
+      // TODO finish this test
```
Comment (denalimarsh) [inline ethereum/test/test_verification.js:79]: I'm assuming this TODO is intentional and will be addressed shortly?

**Review by denalimarsh — APPROVED** [review]
Looks good to me. Much easier to review now, thanks.

File: ethereum/test/test_verification.js (hunk 1 above)
Comment (philipstanislaus) [reply ethereum/test/test_verification.js:79]: Yes, will come in my next PR. Thanks for the review 🙌

**Pull request merged**